model-engine/setup.cfg

```diff
@@ -11,15 +11,19 @@ omit =
     model_engine_server/core/celery/app.py
     model_engine_server/core/docker/ecr.py
     model_engine_server/db/base.py
+    model_engine_server/domain/use_cases/llm_model_endpoint_use_cases.py
     model_engine_server/infra/gateways/abs_file_storage_gateway.py
     model_engine_server/infra/gateways/abs_filesystem_gateway.py
     model_engine_server/infra/gateways/abs_llm_artifact_gateway.py
+    model_engine_server/infra/gateways/asb_inference_autoscaling_metrics_gateway.py
+    model_engine_server/infra/gateways/redis_inference_autoscaling_metrics_gateway.py
     model_engine_server/infra/gateways/resources/asb_queue_endpoint_resource_delegate.py
     model_engine_server/infra/gateways/resources/live_endpoint_resource_gateway.py
     model_engine_server/infra/gateways/resources/k8s_resource_types.py
     model_engine_server/infra/repositories/abs_file_llm_fine_tune_events_repository.py
     model_engine_server/infra/repositories/abs_file_llm_fine_tune_repository.py
     model_engine_server/infra/repositories/acr_docker_repository.py
+    tests/unit/conftest.py
```

Comment:
wait do we need to skip codecoverage for this file? you'd think that this file is one of the most covered files